scripts/superdesk/editor2/add-content.js

```diff
@@ -132,7 +132,7 @@ function AddContentCtrl (scope, element, superdesk, editor, $timeout, config, $q
             addPicture: function() {
                 superdesk.intent('upload', 'media').then(function(images) {
                     $q.all(images.map(function(image) {
```

Comment:
Since these can now be both images and videos, maybe it's best we rename `images` and `image` to `items` or `item` to avoid confusion.